### ISSUE TITLE
When edit file on a remote mounted directory , the `SlimeSend` command lag.

### ISSUE THREAD

**Issue by siuoly** [jpalardy/vim-slime#338]
When I use `sshfs <remote_ip>:<remote_dir> local_dir -p <port>` to mount a remote directory, edit file on the *local_dir*,  and then use *SlimeSend* command to tmux target , the editor is blocked about 2 second.

If I using `vimterminal` as target, *SlimeSend*  is not blocked.

I guess that the reason is that vim load remote file context, or the paste file like `"$HOME/.slime_paste"`.

Is there any recommend solution ?

Now I use `vimterminal` as target and open a tmux session in the `vimterminal`, but I am looking for a better way.

**Comment by jpalardy** [comment]
Hi @siuoly 

An interesting case I hadn't considered 🤔 

It's true that `vimterminal` doesn't use [WritePasteFile](https://github.com/jpalardy/vim-slime/blob/main/autoload/slime.vim#L296-L305), so it wouldn't be subject to this problem. But I'm surprised that `$HOME/.slime_paste` is considered to be on the remote file context…

Can you confirm that? Do you see a `.slime_paste` file on the remote directory? (and do you see your latest paste in it?)

**Comment by siuoly** [comment]
Thanks for response !!

Hmm..., I noticed the pasted file in local directory `$HOME/.slime_paste`, and actually the latest pasted content in it (local).
So this problem is nothing to do with `$HOME/.slime_paste`.

But I wonder is there any  i/o operation on the file? 
Like `:!cat`, `:!ls` ... would cause delay because it access the remote filesystem potentially.

I traced source code (autoload/slime.vim) but got nothing.
